fdbclient/NativeAPI.actor.cpp

```diff
@@ -654,6 +655,19 @@ ACTOR Future<Void> databaseLogger(DatabaseContext* cx) {
 		cx->bgLatencies.clear();
 		cx->bgGranulesPerRequest.clear();
 
+		if (cx->usedAnyChangeFeeds && logTraces) {
```

Comment:
Rather than depending on used change feeds, is there a way to just depend on if blob is enabled?